pages/views/_layouts/base.njk

```diff
@@ -22,7 +22,28 @@
     <link rel="stylesheet" type="text/css" href="{{ '/bundles/localdev.css' | url }}" />
   {% endblock %}
   {% block head %}
-    <script async src="https://cse.google.com/cse.js?cx=3171f866738b34f02"></script>
+    <script async>
+      if (window.location.pathname === '/search.html') {
+        addScript();
+      }
+      
+      if (window.location.pathname === '/') {
+        document.addEventListener('DOMContentLoaded', function() {
+          var searchBar = document.querySelector('.header-search-box');
+          var searchBarShowHandler = function() {
+            addScript();
+            searchBar.removeEventListener('esl:show', searchBarShowHandler)
+          }
+          searchBar.addEventListener('esl:show', searchBarShowHandler);
+        });
+      }
+
+      function addScript() {
```

Comment:
When it's appeared in the right place (our ts sources for pages) the are all helpers for loading will be presented